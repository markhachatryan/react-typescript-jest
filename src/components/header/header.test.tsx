import * as React from "react";
import Header from "./header";
import * as renderer from "react-test-renderer";

test("test for react test", () => {
  const component = renderer.create(<Header handelSubmitSearch={() => {}} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
