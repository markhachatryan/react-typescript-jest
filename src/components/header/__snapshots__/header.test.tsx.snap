// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test for react test 1`] = `
<div
  className="header"
>
  <form
    className="header-search"
    onSubmit={[Function]}
  >
    <input
      onChange={[Function]}
      type="search"
      value=""
    />
    <button
      type="submit"
    >
       Search 
    </button>
  </form>
</div>
`;
